### PR TITLE
Item flag, NoSale, AH category, BaseSell, and stackSize adjustments

### LINKED ISSUE #1778: [AH] [SQL] Some items and/or equipment are missing auction house category
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)

## Branch affected by issue 

`base`

## Steps to reproduce

Place any of these items on auction house:
barrier module 
magniplug 
jug of tantalizing broth 
jug of blackwater broth 
aiwon gauntlets |
rager ledelsens +1
gigantes boots +1
megasco earring

try looking for them on the buy auction house
wont be able to find them
## Expected behavior

Item should show on auction house, but doesn't as their `Ah` column is set to `0`

is the `Ah` column populated manually or does it need a polutils export?

### PR DESCRIPTION
**Pull request by Era-Lusiphur**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

There were some issues with stale PR #2611 by @illzbee and a vocalized desire to split the scope of the PR up. Most of his PR's work with item_basic was updating items with a specific flag to NoSale, along with adding AH categories for some items that were missing in an open issue. I've done the same adjustments here, along with setting additional items to NoSale with similar flags that I noticed.

I also caught a few items with incorrect flags and other values, like Malignance Earring and Yngvi Choker, examining their flags from POLUtils.

There is likely still a lot of cleanup to be done here in the future.

As with @illzbee 's PR, this closes #1778 .

## Steps to test these changes

Co-authored-by: @illzbee

### CODE REVIEW

File: sql/item_basic.sql [hunk 1]
```diff
@@ -13516,13 +13516,13 @@ INSERT INTO `item_basic` VALUES (16190,0,'bloom_buckler','bloom_buckler',1,34820
 INSERT INTO `item_basic` VALUES (16191,0,'utilis_shield','utilis_shield',1,63552,0,1,0);
 INSERT INTO `item_basic` VALUES (16192,0,'ochain','ochain',1,63552,0,1,0);
 INSERT INTO `item_basic` VALUES (16193,0,'ochain','ochain',1,63552,0,1,0);
-INSERT INTO `item_basic` VALUES (16194,0,'ochain','ochain',1,63552,0,0,0);
+INSERT INTO `item_basic` VALUES (16194,0,'ochain','ochain',1,63552,0,1,0);
 INSERT INTO `item_basic` VALUES (16195,0,'aegis','aegis',1,63552,0,1,0);
 INSERT INTO `item_basic` VALUES (16196,0,'aegis','aegis',1,63552,0,1,0);
 INSERT INTO `item_basic` VALUES (16197,0,'aegis','aegis',1,63552,0,1,0);
-INSERT INTO `item_basic` VALUES (16198,0,'aegis','aegis',1,63552,0,0,0);
-INSERT INTO `item_basic` VALUES (16199,0,'ochain','ochain',1,63552,0,0,0);
```
Comment (Xaver-DaRed) [inline sql/item_basic.sql:13524]: you changed the item ID here

Comment (Era-Lusiphur) [reply sql/item_basic.sql:13524]: Corrected.

**Review by Xaver-DaRed — APPROVED** [review]
I'll just have to trust you and make you accountable for this, Lusi.

I obviouly didn't check them all, but the ones I did check seemed correct.

**Comment by TeoTwawki** [comment]
normally I'd insist on breaking this into multiple smaller item batches...But I know where you live. ⚔️

**Pull request merged**